packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts

```diff
@@ -105,9 +105,9 @@ export class EdgelessShapeToolButton extends ToolbarButtonWithMenuMixin<
   }
 
   private _handleShapeClick(shape: DraggableShape) {
-    if (!this.popper) {
-      this._toggleMenu();
-    }
+    // if (!this.popper) {
```

Comment:
comments

Reply:
Removed since it's handled in wrapper's `@click` callback